### DISCUSSION TITLE
ProtheusLibCoreModule

### DISCUSSION THREAD

**Discussion by EdrianGolob** [po-ui/po-angular#1697] [Tire suas dúvidas] [answered]
Boa tarde pessoal.

Tenho uma aplicação em Angular com PO-UI.
Essa aplicação está na versão 13 do Angular.

Ela funciona normalmente no TOTVS Datasul, porém, vi que pra subir no Protheus, precisa fazer o FWCall que usa o ProtheusLibCoreModule.

import { ProtheusLibCoreModule } from '@totvs/protheus-lib-core';

A versão que ele instala é a 14, que requer que o Angular esteja na 14 também. Tem alguma forma de instalar esse Modulo para versão 13? Não queria fazer a atualização da Aplicação ainda, por isso a dúvida.
Ou se tem alguma forma de subir a aplicação do Protheus não utilizando isso.

Obrigado.

Att

**Comment by anliben** [comment, marked as answer]
Boa Tarde @EdrianGolob, tudo bem?

Esta dúvida não e referente ao PO-UI e nosso time não oferece suporte a lib `protheus-lib-core`

Attr.
